.github/workflows/wheels.yml

```diff
@@ -90,6 +90,7 @@ jobs:
   upload:
     needs: [wheels, sdist]
     runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Thanks! Something like this is what I was wanting. I could test this before making the PR as long as I'm not on master now. (You could also make sure it's coming from scikit-hep/iminuit, but now that you have this check, that's enough, at least for me)